### PR TITLE
Update balboa-http integration tests

### PR DESCRIPTION
**Pull request by jacobsi**
Add support for starting a Docker container with a Cassandra instance,
importing the necessary schema and connecting balboa-http to the
instance for the integration tests.

### CODE REVIEW

File: etc/balboa-cassandra.cql [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
--- The first query needs the correct data center to be filled in before this
--- file is executed.
+-- This file only works with the `cassandra-cli` command line utility. That
+-- utility is deprecated and due to be removed from Cassandra in 2.2. This
+-- schema will have to be ported to cqlsh to upgrade to that release.
 --
-
 create keyspace Metrics2012
   with placement_strategy = 'NetworkTopologyStrategy'
-  and strategy_options = {<- FILL IN WITH DATACENTER -> : 3}
```
Comment (TranquilMarmot) [inline etc/balboa-cassandra.cql]: I like this part :)

File: jenkins/integration-tests [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 
+# The balboa-agent integration tests exist but don't yet have the necessary
+# functionality to start up ActiveMQ, so for the moment they are removed from
+# this script, which is run by Jenkins as part of every pull request.
 # Right now the `balboa-agent` integration tests, when executed on Jenkins,
 # hang looking for the ActiveMQ server to remove the queue from during clean
 # up. As an emergency step, I'm disabling those tests for the moment.
 #balboa-agent/src/it/bash/integration-tests
-
-# Disabling the balboa-http tests pending the merge of the branch that makes them go.
```
Comment (TranquilMarmot) [inline jenkins/integration-tests]: I suppose this is the branch that makes them go 😆

**Comment by TranquilMarmot** [comment]
lgtm

File: jenkins/integration-tests (hunk 2 above)
Comment (jacobsi) [inline jenkins/integration-tests]: That's the hope. Seems to work so far.

**Pull request merged**